fecgov/openFEC
# Remove deprecated '/communication-costs/' endpoint (replaced by '/communication_costs/')

## Summary (required)

- Resolves #4546

Remove deprecated `/communication-costs/` endpoint

## Reviewers
One approval requested, others welcome

## How to test the changes locally

- Review code, make sure steps are followed from original issue: https://github.com/fecgov/openFEC/issues/4546
- Check out this branch, run it locally
- Make sure `/communication-costs/` doesn't appear in Swagger docs: http://localhost:5000/developers/#/communication%20cost
- A test query to  `/communication-costs/` 404's: http://localhost:5000/v1/communication-costs/
- A test query to  `/communication_costs/` works: http://localhost:5000/v1/communication_costs/

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Removing deprecated endpoint
